### PR TITLE
Fixed some typos.

### PR DESCRIPTION
**Pull request by irfanevrens**

### CODE REVIEW

File: Slim/Http/Headers.php [hunk 1]
```diff
@@ -9,7 +9,6 @@
 namespace Slim\Http;
 
 use Slim\Collection;
-use Slim\Http\Environment;
```
Comment (akrabat) [inline Slim/Http/Headers.php:12]: We use this for type hinting on line 51 of this file:

``` php
public static function createFromEnvironment(Environment $environment)
```

**Comment by akrabat** [comment]
You delete `Slim/Interfaces/ContainerInterface.php` however, I can't find this file in https://github.com/slimphp/Slim/tree/3.x/Slim/Interfaces ?

File: Slim/Http/Headers.php (hunk 1 above)
Comment (irfanevrens) [inline Slim/Http/Headers.php:12]: Environment and Headers are in the same namespace, Slim\Http.

Comment (akrabat) [inline Slim/Http/Headers.php:12]: Doh! So they are :)

**Comment by irfanevrens** [comment]
Added on 43b693480028d48f387a620a34a9cafdfebcc767 then removed on 02b629c63632c02c7b0a44b340130bd85791f4eb. It was my local file.

**Comment by akrabat** [comment]
Thanks!

**Pull request merged**